api/apps/geoprocessing/src/modules/planning-area/planning-area.dto.spec.ts

```diff
@@ -0,0 +1,117 @@
+import { PlanningAreaDto } from './planning-area.dto';
+
+const input = () => ({
```

Comment:
```suggestion
const input = (): GeoJSON & {puSizes?: ....}  => ({
```
sqrt(💊) as `as const` seems to be awkward workaround for GeoJSON type 🤔 nevertheless, just specs.  

(but I still think that those additional metadata should go into `properties` of a feature)

Reply:
it will be a little more verbose but ok :)

Reply:
fixed